### ISSUE TITLE
When integrating with an existing app we had few flexibility in the routing

### ISSUE THREAD

**Issue by cesarob** [wagtail/wagtail#4342]
Found a bug? Please fill out the sections below. 👍

### Issue Summary

The integration with an existing app was quite straightforward due the good documentation but we found it difficult to tune the routing.
We added a blog to an existent app. In case we have chosen `mydomain.com/blog` as the root everything should go fine, but the problem is we wanted to reverve the root in case we wanted to add more content though the cms. So now our root of the blog would be:

`mydomain.com/content/blog`

The problem here is that we want the user access to the blog as `mydomain.com/blog` so in oder to do that we need to add a new route `mydomain.com/blog` and : 

+ add a view that redirects to the proper one:

```
def blog_view(request, path):
    return wagtail.wagtailcore.views.serve(request, path='blog/' + path)
```

+ When mananing urls in the templates we need to call our own Page method in order to have the right url:

```
def get_url(self, request=None, current_site=None):
        return self.url.replace('/{}'.format('content'), '')
```

What we just need is the possibility of having more than one root. Is that possible? If not I think it would be an interesting feature when integrating with an existent app.

### Steps to Reproduce

It is not a bug.

### Technical details
* Wagtail version: 1.13.1

**Comment by allcaps** [comment]
I've got a hard time understanding what you actually want to do. Note that this issue tracker is for bugs and features on Wagtail itself. It is not an support channel. Good places for support are listed here: http://docs.wagtail.io/en/v2.0/support.html

For your problem, put your blog view route before wagtail:

```
from blogapp import urls as blog_urls

urlpatterns = [
    url(r'^blog/', include(blog_urls)),
    url(r'', include(wagtail_urls)),
]
```

**Comment by cesarob** [comment]
Hi there,
I am not requesting support as we already solved the problem. Just I wanted to mention that for us the integration was a bit tricky (being the initial usage of wagtail a blog).

Maybe I am wrong but you just support one root in the wagtail routing. Don't you?

**Comment by allcaps** [comment]
Wagtail allows mapping a “site” (hostname or domain) to any node in the tree, using that node as the site’s root. http://docs.wagtail.io/en/v0.7/core_components/sites.html

**Comment by cesarob** [comment]
Yeah, I saw it. A possible solution is to use subdomains. Right now we aren't using them.

As we have already discussed it, if you think it is not an interesting feature I think you close the issue ;).

Thanks,
